native/swift/Sources/wordpress-api/WordPressAPI.swift

```diff
@@ -68,11 +68,8 @@ public struct WordPressAPI {
     }
 
     package func perform(request: WpNetworkRequest) async throws -> WpNetworkResponse {
-        try await withCheckedThrowingContinuation { continuation in
-            self.perform(request: request) { result in
-                continuation.resume(with: result)
-            }
-        }
+        let (data, response) = try await self.urlSession.data(for: request.asURLRequest())
```

Comment:
Swift Foundation has feature parity across platforms as of Swift 6, so we should just be able to use the async/await stuff everywhere now?